extras/label_configs/carrots

```diff
@@ -1,5 +1,5 @@
 # Remove Github defaults
--import:clear-defaults
+-delete:all
```

Comment:
So, I think it is an edge case, as you're really supposed to use this lib right when a repo starts off.

If you're calling this in any other time than that, we shouldn't be deleting labels as we could lose important tags.